Makefile

```diff
@@ -0,0 +1,21 @@
+lint:
+	@echo
+	ruff .
+	@echo
+	black --check --diff --color .
+	@echo
+	pip-audit
+
+format:
+	ruff --silent --exit-zero --fix .
+	black .
+
+precommit:
+	make lint
+	make format
+
+venv:
+	python3 -m venv ecout_env
```

Comment:
python or python3?

Reply:
I think python3 is preferred since we are working with Python 3.x.x

Reply:
Could somehow use either? For me I don't have python3 alias only python in my PATH.

Reply:
Yeah, I can change it to python instead of python3.

Reply:
Done.